stern/watch.go

```diff
@@ -107,25 +111,29 @@ func Watch(ctx context.Context, i v1.PodInterface, podFilter *regexp.Regexp, exc
 						}
 					}
 				case watch.Deleted:
-					var containers []corev1.Container
-					containers = append(containers, pod.Spec.Containers...)
+					var containerNames []string
```

Comment:
Container and InitContainer is [`k8s.io/api/core/v1.Container`](https://github.com/kubernetes/api/blob/v0.21.0/core/v1/types.go#L2201), but EphemeralContainer is
[`k8s.io/api/core/v1.EphemeralContainer`](https://github.com/kubernetes/api/blob/v0.21.0/core/v1/types.go#L3416).
So I changed receiving container name directly.